cet release notes for 5.22.0 (#1890)
harness/developer-hub: release-notes/continuous-error-tracking.md

  <TabItem value="Fixed issues">
```

> - The CET Agent list is failing to load correctly when running in SMP installations with Postgres. (CET-1279)  

Reviewer: ```suggestion
- In SMP installations with Postgres, the CET Agent list does not load correctly. (CET-1279)  
```

>     The issue has been resolved. The CET Agent list now loads properly in SMP installations with Postgres.

Reviewer: How about "A code enhancement has fixed this issue."?
